source/locales.tex

```diff
@@ -5073,6 +5063,11 @@
 \tcode{ranges::range_value_t<text_encoding::aliases_view>} and
 \tcode{ranges::range_reference_t<text_encoding::aliases_view>}
 denote \tcode{const char*}.
+
+\pnum
+%FIXME: Is this supposed to be a remark or a requirement? Same above?
```

Comment:
Can we remove this FIXME comment? These are statements of fact, so requirements on implementations.

_Remarks_ and _Requirements:_ only apply to functions and function templates, not types, so using either of those would be wrong here. There is nothing to fix here, because this is the correct way to specify properties of a library type (see e.g. [text.encoding.overview] p2, and [pairs.pair] p1-4, and dozens and dozens of other places in the library clauses).

Reply:
Yes, we can remove those. Care to send a PR, or just commit this directly.

Reply:
I'm sending a PR in the same subclause so will remove it as part of that.